recipes/libgd/all/test_package/conanfile.py

```diff
@@ -0,0 +1,20 @@
+import os
+
+from conans import ConanFile, CMake, tools
+
+
+class LibgdTestConan(ConanFile):
+    settings = "os", "compiler", "build_type", "arch"
+    generators = "cmake"
+
+    def build(self):
+        cmake = CMake(self)
+        # Current dir is "test_package/build/<build_id>" and CMakeLists.txt is
+        # in "test_package"
+        cmake.configure()
+        cmake.build()
+
+    def test(self):
+        if not tools.cross_building(self.settings):
+            os.chdir("bin")
+            self.run(".%slibgd_test" % os.sep, run_environment=True)
```

Comment:
```suggestion
            bin_path = os.path.join("bin", "libgd_test")
            self.run(bin_path, run_environment=True)
```

Not because your implementation is wrong, but because there is a pattern which keeps easier to remember and review.